src/main/java/pl/nogacz/checkers/board/Board.java

```diff
@@ -78,12 +79,75 @@ private void addStartPawn() {
             Design.addPawn(entry.getKey(), entry.getValue());
         }
     }
+    public  boolean getisBlackRound(){
+        if(isBlackRound)
+        return true;

```

Comment:
fix indentation and enclose ıf and else body wıth parenthesis

Reply:
accomplished